enhancement/planner.py

```
@@ -0,0 +1,334 @@
+    
+    def create_intent(
+        self,
+        book_id: str,
+        chapter_num: int,
+        must_keep: List[str],
+        must_avoid: List[str],
+        emotional_tone: str = "neutral",
+        chapter_goal: str = "",
+        word_count_target: int = 14000,
+        word_count_range: tuple = (12600, 15400),
+        related_hooks: List[str] = None,
```

Ruff (0.15.10)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)